lib/Bindings/Python/CIRCTModule.cpp

```diff
@@ -12,14 +12,22 @@
 #include "circt-c/Dialect/ESI.h"
 #include "circt-c/Dialect/RTL.h"
 #include "circt-c/Dialect/SV.h"
+#include "circt-c/ExportVerilog.h"
 #include "mlir-c/Bindings/Python/Interop.h"
 #include "mlir-c/Registration.h"
 
+#include "llvm-c/ErrorHandling.h"
+
+#include "PybindUtils.h"
 #include <pybind11/pybind11.h>
 namespace py = pybind11;
 
+static void registerPasses() { registerSVPasses(); }
+
 PYBIND11_MODULE(_circt, m) {
   m.doc() = "CIRCT Python Native Extension";
+  registerPasses();
```

Comment:
Registering the passes when the module is imported sounds nice, should we do the same for registering dialects instead of having a separate `register_dialects` hook?

Eventually, we might not want to register all dialects/passes at once, but I guess we can cross that bridge when we come to it.

Reply:
I thought about that, but registering a dialect requires an `MLIRContext`, whereas registering passes does not.

Reply:
Ah ok that makes sense to me.